### ISSUE TITLE
issues with import functions

### ISSUE THREAD

**Issue by kaizadp** [kaizadp/nmrrr#45]
@bpbond 

Our functions are now unable to import the files in correctly, and I'm not sure why.

This error occurs only in the `shorten_data_files` branch, and not in the main one. All I did was delete (move) a lot of the files. Where we previously had 9-10 spectra/peak files, we now have only 2. [Files here](https://github.com/bpbond/nmrrr/tree/shorten_data_files/inst/extdata/kfp_hysteresis).

It's throwing up [CMD-check errors](https://github.com/bpbond/nmrrr/actions/runs/3348344822/jobs/5547309639)

Two problems:
1. peaks files: does not recognize the "9-column groups" for the MNova peaks with the "multiple columns" selection (see error log)

![image](https://user-images.githubusercontent.com/50244730/198744665-48162395-f0dc-4c00-8c45-27a46f64a647.png)

2. spectra files: because of the weird tab vs. comma delim nonsense of MNova, it is not correctly splitting the data into the columns (see below)

Tried testing the code in the `scratchpad` file, it imports the files wrong.  
https://github.com/bpbond/nmrrr/blob/bfc997d3198be7eab6befbc64a8fd789bdbb5b7f/old/scratchpad.R#L19-L21

![image](https://user-images.githubusercontent.com/50244730/198737683-de975467-e57e-4a8e-a79b-8b321836daae.png)

**Comment by kaizadp** [comment]
also -- our spectra files were really big for some of the examples (amp and meb). I tried really hard, but was unable to bring down the file size significantly. So I moved those to the `old` folder for now, so they are not part of the package. Can we leave them on the GitHub repo, so users can play with these datasets if they want? Or if it's not in the package, it should not be in the repo?

**Comment by bpbond** [comment]
@kaizadp Did you perhaps change the example code incorrectly in 9a13165 ?

On this branch, the two files in `inst/extdata/kfp_hysteresis/spectra_mnova` are `29.csv` and `34.csv`, which are both two-column, no header. But the example is
```r
sdir <- system.file("extdata", "kfp_hysteresis", "spectra_mnova", package = "nmrrr")
nmr_import_peaks(path = sdir, method = "multiple columns")
```
...and "multiple columns" is expecting to see nine-column groups. 😮 

The old `nmr_import_peaks` example (on the master branch) is 
```r
pdir <- system.file("extdata", "meb_burn", "peaks_topspin", package = "nmrrr")
nmr_import_peaks(path = pdir, method = "topspin")
```

**Comment by bpbond** [comment]
Also, we can look/talk about file sizes. But it's OK to have extra stuff in the repo that's not in the package, I think.

**Comment by kaizadp** [comment]
UGH. 

1. The issue with `nmr_import_peaks` examples was because of wrong file paths (sigh). Now fixed. 
9a0c124a339456639d6425e3cb370538369c59e7
6c3113e407b892a20fef8963876a24eeca38fef7

2. The spectra files were somehow messed up when I tried to shorten them by deleting rows outside of 0-10 ppm. Something to do with how the files were saved from MNova? Fixed those by re-exporting only 0-10 ppm data directly from MNova.
f6f3e78c3a57e68eba714e16199093eed6e69d39
4244917c76c291dcdd7f0651e9c7741441b57f80
